rpc/wrtc_peer.go

```diff
@@ -236,6 +233,16 @@ func newPeerConnectionForServer(
 	return peerConn, dataChannel, nil
 }
 
+// PeerRole identifies which role of a Client/Server relationship a peer is assuming.
+type PeerRole bool
```

Comment:
I thought this was a helpful addition to `ConfigureForRenegotiation` since we differentiate the behavior, but happy to remove it if it's too much fluff.